app/assets/javascripts/kassi.js

```diff
@@ -722,17 +722,28 @@ function initialize_accept_transaction_form(
   }
 }
 
-function updateSellerGetsValue(priceInputSelector, youWillGetSelector, currencySelector, communityCommissionPercentage, minCommission) {
-  $display = $(youWillGetSelector);
+function updateSellerGetsValue(priceInputSelector, displayTargetSelector, currencySelector, communityCommissionPercentage, minCommission, showReversed) {
```

Comment:
Normally I'd agree but knowing some of the plans we have for mid-term future with rethinking the UI I don't think it's worth the time and effort to clean up the existing JS since it's likely to be thrown out anyway.